src/js/window.js

```diff
@@ -33,7 +33,7 @@ export default class WindowHandler {
     if (this.fileHandler.isSupported) {
       $('#send-file').click((e) => {
         e.preventDefault();
-        $('#fileInput').trigger('click');
+        document.getElementById('fileInput').click();
```

Comment:
Just testing different ways to attempt to trigger the file input on safari, we can actually revert it back.